server.elm

```diff
@@ -1784,8 +2014,13 @@ editionscript = """
       modifyMenuDiv.innerHTML = "";
       let modifyMenuIconsDiv = el("div", {"class":"modify-menu-icons"});
       let interactionDiv = el("div", {"class": "information"});
+      let domSelector = el("div", {"class": "dom-selector"}); // create dom selector interface
+      
+      modifyMenuDiv.append(domSelector);
       modifyMenuDiv.append(modifyMenuIconsDiv);
       modifyMenuDiv.append(interactionDiv);
+
+      let domSelectorDiv = document.querySelector(".dom-selector");
```

Comment:
Remove this line. `domSelectorDiv` and `domSelector` are the same node. No need to query it. Rename `domSelectorDiv` to `domSelector`